src/stcal/jump/jump.py

```diff
@@ -116,6 +134,9 @@ def detect_jumps(frames_per_group, data, gdq, pdq, err,
     data *= gain_2d
     err *= gain_2d
     readnoise_2d *= gain_2d
+    # also apply to the after_jump thresholds
+    after_jump_flag_e1 = after_jump_flag_dn1 * gain_2d
```

Comment:
Given that `gain_2d` is a 2D array of gain values, does that mean `after_jump_flag_e[12]` are also now full 2D arrays?

Reply:
Yes.

Reply:
Update the docstring for find_crs.